test/indexer/test_hca_indexer.py

```diff
@@ -551,6 +560,93 @@ def test_organoid_priority(self):
                          inner_cell_suspensions)
 
 
+class TestValidNotificationRequests(LocalAppTestCase):
+
+    @classmethod
+    def lambda_name(cls) -> str:
+        return "indexer"
+
+    @mock_sts
+    @mock_sqs
+    def test_post_notification_endpoint(self):
+        self._create_mock_notify_queue()
+        body = {
+            'match': {
+                'bundle_uuid': 'bb2365b9-5a5b-436f-92e3-4fc6d86a9efd',
+                'bundle_version': '2018-03-28T13:55:26.044Z'
+            }
+        }
+        for endpoint in ['/', '/delete']:
+            with self.subTest(endpoint=endpoint):
+                response = self._test(body, endpoint)
+                self.assertEqual(202, response.status_code)
+                self.assertEqual('', response.text)
+
+    @mock_sts
+    @mock_sqs
+    def test_invalid_notifications(self):
+        for endpoint in ['/', '/delete']:
+            with self.subTest(endpoint=endpoint):
+                bodys = {
```

Comment:
```suggestion
                bodies = {
```

Reply:
Can we pull this dict literal to the beginning of the method, outside of the loop and any subTest contexts?